frontEnd/src/LoginPage.jsx

```diff
@@ -3,6 +3,19 @@ import "./App.css";
 function LoginPage() {
   const itemsPage = `${import.meta.env.VITE_BASE_URL}items`;
   const mainPage = `${import.meta.env.VITE_BASE_URL}`;
+  const backendUrl = import.meta.env.VITE_BACKEND;
+
+  const loginFunc = async (event) => {
+    event.preventDefault();
+    const formData = new FormData(document.getElementById("login"));
+    let readableData = Object.fromEntries(formData);
+    const response = await fetch(`${backendUrl}login`, {
```

Comment:
i think `${backendUrl}/login` should also work even if your backendUrl ends with a backslash, it'll be more robust in case somehow the trailing backslash gets deleted